src/package/pywinrt/projection/AzurePipelinesTemplates/steps-build-projection.yml

```diff
@@ -0,0 +1,85 @@
+parameters: # defaults for any parameters that aren't specified
+  architecture: 'x86'
+  pythonVersionSpec: '3.7.x'
+  projectionType: 'partial'
+
+steps:
+- task: DownloadBuildArtifacts@0 
+  inputs: 
+    buildType: 'current'
+    downloadType: single
+    artifactName: 'pywinrt'
+    downloadPath: '$(Build.ArtifactStagingDirectory)'
+
+# I have been completely unable to wire up queue-time variables to template parameters so that they work with conditional insertion
+# so instead, I'm passing the projectionType value into the script and switching between full/partial via a script if statement
+- script: |
+    echo %ProjectionType%
+    IF %ProjectionType%==full (
+        $(Build.ArtifactStagingDirectory)/pywinrt/pywinrt.exe -input 10.0.17763.0 -output src/package/pywinrt/projection/pywinrt -verbose -include Windows. -exclude Windows.UI.Composition -exclude Windows.UI.Xaml
```

Comment:
>10.0.17763.0 [](start = 69, length = 12)

consider creating an SdkVersion parameter

Reply:
yeah, I thought about that. Given the issues I had getting the full vs. partial build parameter working, I am not going to tackle SDK version as a param right now